cpp_test_suite/new_tests/cxx_group.cpp

```diff
@@ -827,6 +827,39 @@ class GroupTestSuite: public CxxTest::TestSuite
 		CxxTest::TangoPrinter::restore_unset("double_attr_value");
 	}
 
+	/* Verifies that a group can contain devices from a remote TANGO_HOST
+	 * (a Tango instance different from client's default TANGO_HOST).
+	 * An issue was reported when resolving names containing wildcards. */
+
+	void test_use_devices_from_remote_tango_host()
+	{
+		const std::string original_tango_host = std::getenv("TANGO_HOST");
+		const std::string external_tango_host = std::getenv("TANGO_HOST2");
+
+		const bool force_update = true;
+
+		TS_ASSERT_EQUALS(0, setenv("TANGO_HOST", external_tango_host.c_str(), force_update));
+		ApiUtil::instance()->cleanup();
+
+		Group group("group");
+		group.add("tango://" + original_tango_host + "/" + device1_name + "*");
+
+		GroupCmdReplyList command_results = group.command_inout("State");
+		GroupCmdReply command_result = command_results[0];
+
+		TS_ASSERT(not command_results.has_failed());
```

Comment:
Question:
Do we want to use alternative operator names like `not` in tango? See https://en.cppreference.com/w/cpp/language/operator_alternative.

Reply:
This can cause some compilation errors when using some old compilers on Windows. But since we don't compile the tests on Windows, this should not be an issue here.
In this specific case, the code is even more clear when using this alternative operator.

Reply:
@t-b are you ok with leaving it as it is now?